### ISSUE TITLE
set_elements: EELS and EDS spectrum

### ISSUE THREAD

**Issue by magnunor** [hyperspy/hyperspy#725]
Currently EDS spectrum has both add_elements and set_elements, but EELS spectrum has only add_elements.

Should we add set_elements to EELS spectrum as well? To make it more consistent between the two spectrum types?

**Comment by francisco-dlp** [comment]
Good point. May it be possible to have superclass of EELS and EDS (AnalyticalSpectrum?) that includes these methods which are common to these signals?

Giving that the main focus of HyperSpy 0.9 will be curve fitting, I'll add this to the 0.9 milestone.
